.circleci/config.yml

```diff
@@ -255,12 +262,26 @@ jobs:
               make build-linux
               make localnet-start
               ./scripts/localnet-blocks-test.sh 40 5 10 localhost
-
+  deploy_docs:
+    <<: *docs_deploy
+    steps:
+      - checkout
+      - run:
+          name: Trigger website build
+          command: |
+            export LAST_COMMIT=`git rev-parse HEAD`
+            export DOCS_COMMIT=`git log -1 --format=format:%H --full-diff docs`
+            if [[ $DOCS_COMMIT == $LAST_COMMIT ]]; then
+              chamber exec cosmos-sdk -- start_website_build
```

Comment:
Guessing this binary is in the docker image above, but it would be nice to see some docs on this process. Maybe they are in a different repo?